scss/titles.scss

```diff
@@ -1,6 +1,7 @@
 .title {
   color: $training-black;  
   font-size: $text-big;
+  margin: 35px 0;
```

Comment:
I think it would be better to use the padding in the parent for the top margin and use only margin bottom to separate this element from its siblings (the techs)